packages/react-native/src/primitives/Button/styles.ts

```diff
@@ -1,13 +1,42 @@
-import { StyleSheet } from 'react-native';
+import { StyleSheet, TextStyle } from 'react-native';
 
 import { StrictTheme } from '../../theme';
 import { ButtonStyles } from './types';
 
 export const getThemedStyles = (theme: StrictTheme): ButtonStyles => {
-  const { components, opacities } = theme.tokens;
+  const { colors, components, fontSizes, fontWeights, opacities, space } =
+    theme.tokens;
 
   return StyleSheet.create({
+    container: {
+      padding: space.large,
+      ...components?.button.container,
+    },
+    containerPrimary: {
+      backgroundColor: colors.brand.primary[80],
+      ...components?.button.containerPrimary,
+    },
+    containerSecondary: {
+      backgroundColor: colors.background.primary,
+      ...components?.button.containerSecondary,
+    },
+    disabled: {
+      opacity: opacities[60],
+      ...components?.button.disabled,
+    },
     pressed: { opacity: opacities[60], ...components?.button.pressed },
-    text: { alignSelf: 'center', ...components?.button.text },
+    text: {
+      alignSelf: 'center',
+    },
+    textPrimary: {
+      color: colors.white,
+      fontSize: fontSizes.medium,
```

Comment:
Should we use the same `fontSize` for both variants or have a specific value for the `secondary` variant?

Reply:
We could use a specific size. I thought that making them slightly different sizes looked right for our use case in the RNA.

Reply:
RNA use case and default use case are separate. Although the RNA styling should be aligned with our default components, ultimately it is a consumer of the base UI and not the solo use case for the base UI